src/CandidateCard.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+const voteURL = 'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:8080/vote';
+function CandidateCard({candidate, index, cookies}) {
+    const handleVoteCount= (candidate)=> {
+        axios({
+            method: 'get', url: `${voteURL}?id=${candidate.id}`,
+            headers: {
+                Authorization: cookies.token
+            }
+        })
+        .then (() => {
+            alert(candidate.name + '님에게 투표 완료!');
+        })
+        .catch (() => {
+            alert('투표 실패');
+        })
+    }
+    return(    
+        <EachCandidate>
+            <CandidateRank>{index+1}위 </CandidateRank>
+            <CandidateName>{candidate.name}</CandidateName>
+            <CandidateVoteCount>[{candidate.voteCount}] 표</CandidateVoteCount>
+            <VoteButton onClick={()=>handleVoteCount(candidate)}>투표</VoteButton>
```

Comment:
candidate가 렌더링 스코프 내부에 있으니까 함수 정의 시 인자가 없는 함수로 바꾸고,

```suggestion
            <VoteButton onClick={handleVoteCount}>투표</VoteButton>
```